examples/datepicker/datepicker.html

```diff
@@ -0,0 +1,820 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="utf-8" />
+<title>Date Picker Example Using ARIA 1.1 Combobox | WAI-ARIA Authoring Practices 1.1</title>
+
+<!--  Core js and css shared by all examples; do not modify when using this template. -->
+<link rel="stylesheet" href="https://www.w3.org/StyleSheets/TR/2016/base.css">
+<link rel="stylesheet" href="../css/core.css">
+<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.1.0/css/all.css" integrity="sha384-lKuwvrZot6UHsBSfcMvOkWwlCMgc0TaWr+30HWe3a4ltaBwTZhyTEggF5tJv8tbt" crossorigin="anonymous">
+
+<script src="../js/examples.js" type="text/javascript"></script>
+<script src="../js/highlight.pack.js"></script>
+<script src="../js/app.js"></script>
+
+<link href="css/datepicker.css" rel="stylesheet">
+<script src="js/dateInput.js" type="text/javascript"></script>
+<script src="js/datepickerDay.js" type="text/javascript"></script>
+<script src="js/datepicker.js" type="text/javascript"></script>
+</head>
+<body>
+  <nav aria-label="Related Links" class="feedback">
+    <ul>
+      <li><a href="../../#browser_and_AT_support">Browser and Assistive Technology Support</a></li>
+      <li><a href="https://github.com/w3c/aria-practices/issues/new">Report Issue</a></li>
+      <li><a href="https://github.com/w3c/aria-practices/projects/14">Related Issues</a></li>
+      <li><a href="../../#datepicker">Design Pattern</a></li>
+    </ul>
+  </nav>
+  <main>
+    <h1>Date Picker Example Using ARIA 1.1 Combobox</h1>
+    <p>
+      <strong>NOTE:</strong> This example page is work in progress.
+      Please provide feedback in
+      <a href="https://github.com/w3c/aria-practices/issues/34">issue 34.</a>
+    </p>
+    <p>
+      The date picker is an example of the <a href="../../#combobox"><code>combobox</code> design pattern</a> that opens a dialog box.
+      The date picker dialog box in this example is opened when keyboard focus is moved to the text box and a <kbd>down arrow</kbd> key pressed or clicking on the calendar button.
+      The date picker dialog uses a <code>grid</code> pattern to show and select a date using the cursor keys.  Additional buttons in the dialog box can be used for changing the month and year shown in the grid.
+    </p>
+  <section>
+    <h2 id="ex_label">Example</h2>
+
+    <div role="separator" id="ex_start_sep" aria-labelledby="ex_start_sep ex_label" aria-label="Start of"></div>
+    <div id="example">
+
+      <div id="myDatepicker"
+           class="datepicker"
+           role="combobox"
+           aria-expanded="false"
+           aria-haspopup="dialog"
+           aria-owns="myDatepickerDialog"
+           aria-label="Date">
+
+        <div class="date">
+          <label for="id-date-1">Date</label>
+
+          <input type="text"
+                placeholder="mm/dd/yyyy"
+                id="id-date-1"
+                aria-expanded="false"
```

Comment:
As specified in 1.1 combo pattern, please remove aria-expanded from this input and add aria-autocomplete="none"